tools-local/tasks/mobile.tasks/AndroidAppBuilder/Templates/monodroid.c

```diff
@@ -138,9 +138,9 @@ mono_mobile_runtime_init (void)
 {
     // uncomment for debug output:
     //
-    // setenv ("XUNIT_VERBOSE", "true", true);
-    // setenv ("MONO_LOG_LEVEL", "debug", true);
-    // setenv ("MONO_LOG_MASK", "all", true);
+    setenv ("XUNIT_VERBOSE", "true", true);
+    setenv ("MONO_LOG_LEVEL", "debug", true);
+    setenv ("MONO_LOG_MASK", "all", true);
```

Comment:
it'll ddos CI 😄

Reply:
Will revert this. It is for debugging temporarily.